npm/postinstall.js

```diff
@@ -0,0 +1,52 @@
+const stat = require("fs").stat
```

Comment:
I don't think you should use const here. node 4 (actually older V8 version) required "use strict" mode when using const.